modules/client/src/node.ts

```diff
@@ -114,15 +110,14 @@ export class NodeApiClient implements INodeApiClient {
     });
   }
 
-  public async resolveLinkedTransfer(
-    paymentId: string,
-    preImage: string,
-    amount: string,
-    assetId: string,
-  ): Promise<void> {
+  public async fetchLinkedTransfer(paymentId: string): Promise<any> {
+    return await this.send(`transfer.fetch-linked.${this.userPublicIdentifier}`, {
+      paymentId,
+    });
+  }
```

Comment:
New client method added: get link payment details (eg amount, assetId, redemption status) given the paymentId